core/src/components/fab-button/fab-button.tsx

```diff
@@ -152,8 +153,28 @@ export class FabButton implements ComponentInterface, AnchorInterface, ButtonInt
     this.inheritedAttributes = inheritAriaAttributes(this.el);
   }
 
+  /**
+   * Get the icon to use for the close icon.
+   * If an icon is set on the component, use that.
+   * Otherwise, use the icon set in the config.
+   * If no icon is set in the config, use the default icon.
+   *
+   * @internal
+   * @returns {string} The icon to use for the close icon.
+   */
+  get fabButtonCloseIcon(): string {
```

Comment:
~ This could probably be minimized to:

```ts
get fabButtonCloseIcon(): string {
  return this.closeIcon ?? config.get('fabButtonCloseIcon', close);
}
```

Since it isn't reused in multiple areas I'd recommend moving it into the `render` function:

```ts
const fabButtonCloseIcon = this.closeIcon ?? config.get('fabButtonCloseIcon', close);
```

Avoids the need for a getter, large documentation block and is 1 line of code.